src/main/java/de/tum/in/www1/artemis/web/rest/programming/ProgrammingExerciseExportImportResource.java

```diff
@@ -73,6 +73,8 @@
 import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
 import de.tum.in.www1.artemis.web.rest.errors.ConflictException;
 import de.tum.in.www1.artemis.web.rest.errors.EntityNotFoundException;
+import de.tum.in.www1.artemis.web.rest.errors.HttpStatusException;
+import de.tum.in.www1.artemis.web.rest.errors.InternalServerErrorAlertException;
```

Comment:
Seems out of scope to me, since this would require adding translations and therefore understanding which errors can actually occur there and might result in adjusting the related codebase (in regards of throwing errors)